src/components/DatePicker/DatePickInput/DatePickInput.tsx

```diff
@@ -1,5 +1,6 @@
 import dayjs from 'dayjs';
 import * as React from 'react';
+import omit from 'lodash/omit';
 import { DayPickerInputProps } from 'react-day-picker';
 import useTheme from '../../../hooks/useTheme';
```

Comment:
Is a blank line required above this one, to separate the imports?

Reply:
not quite sure what do you mean here? can you send an example?

Reply:
maybe he is referring to a styling that separates imports from external libraries, internal etc?

Reply:
actually i have such thing in my notes to implement to all projects. But as doing this manually is actually us shooting at our feet i will import a solution to do this with a rule